django/template/defaulttags.py

```diff
@@ -94,7 +94,7 @@ def render(self, context):
         from pprint import pformat
         output = [force_text(pformat(val)) for val in context]
         output.append('\n\n')
-        output.append(pformat(sys.modules))
+        output.append(force_text(pformat(sys.modules)))
```

Comment:
What would the code look like? I didn't look into this very much... just enough to fix the failures.